.storybook/preview.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { Preview } from '@storybook/react';
-import { I18nProvider } from '../packages/react/src/index';
+import { GrunnmurenProvider } from '../packages/react/src/index';
```

Comment:
Her kan vi vel omitte `index` i pathen?
```suggestion
import { GrunnmurenProvider } from '../packages/react/src';
```